samples/README.adoc

```diff
@@ -16,6 +16,12 @@
 - limitations under the License.
 -
 ////
+ifndef::performance_samplespath[:performance_samplespath: ./performance/]
+ifndef::extenstion_samplespath[:extenstion_samplespath: ./extensions/]
```

Comment:
Is that a typo: `extenstion_samplespath`?

Reply:
Indeed. Thanks for noticing!